backend/app/app/api/api_v1/endpoints/ue_movement.py

```diff
@@ -514,53 +325,65 @@ def state_movement(
     """
     return {"running": retrieve_ue_state(supi, current_user.id)}
 
+
 @router.get("/state-ues", status_code=200)
 def state_ues(
     current_user: models.User = Depends(deps.get_current_active_user),
 ) -> Any:
     """
     Get the state
     """
-    return ues
+    with db_context() as db:
```

Comment:
You can use a fastapi depedency here like other endpoints and let fastapi manage the lifecycle of the connection.